package.json

```diff
@@ -17,6 +17,9 @@
     "start": "yarn build && webpack && babel-node example/server.js",
     "prepublish": "yarn build"
   },
+  "dependencies": {
+    "prop-types": "^15.5.7"
```

Comment:
Can you open up the range more to include React 14, 15, and 16?

Reply:
Sure! I just followed Facebook's instructions blindly ;) Which range would you like here? Or is it part of my job to find out?

Reply:
`"^0.14.0 || ^15.0.0 || ^16.0.0"`

Reply:
As far as I can tell, these are the published versions:

```
❯ npm show prop-types versions
 
[ '0.1.0',
  '0.2.0',
  '15.5.0-alpha.0',
  '15.5.0',
  '15.5.1',
  '15.5.2',
  '15.5.3',
  '15.5.4',
  '15.5.5',
  '15.5.6',
  '15.5.7-alpha',
  '15.5.7-alpha.1',
  '15.5.7',
  '15.5.8',
  '15.5.9',
  '15.5.10',
  '15.6.0' ]
```

Reply:
Ah, figured prop-types was being kept in sync with react